CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
   - Enhancements:
     - Use ravel instead of flat in KullbackLeibler numba backend (#1874)
     - Upgrade Python wrapper (#1873, #1875)
+    - New unit tests for operators and functions to check for in place errors and the behaviour of `out` (#1805)
```

Comment:
You also added checks on output datacontainer passed to processors to ensure correct dtype and size. (not geometry I guess?)

You can add things from the same PR to multiple changelog sections to highlight the relevant section. Maybe this should be under `Testing:`

Reply:
Ok I've added this too